src/nodedef.cpp

```diff
@@ -618,6 +621,11 @@ void ContentFeatures::deSerialize(std::istream &is)
 		if (is.eof())
 			throw SerializationError("");
 		alpha = static_cast<enum AlphaMode>(tmp);
+
+		u8 tmp_move_resistance = readU8(is);
+		if (is.eof())
+			throw SerializationError("");
+		move_resistance = tmp_move_resistance;
```

Comment:
What happens if a new client (5.5) connects to an old server (5.4) - the server would not send `move_resistance`, right? Wouldn't `move_resistance` of liquids be `0` then?